lib/stack/NodeBuildInterpreter.ts

```diff
@@ -71,6 +79,33 @@ export interface NodeDeliveryOptions {
     configureTestGoal?: (testGoal: GoalWithFulfillment) => void;
 }
 
+const HasTypescript: PushTest = pushTest("hasTypescript", async p => {
```

Comment:
that should probably use the `hasDependency` method from `nodeScanner.ts` instead of re-implementing the logic here.